### PR TITLE
Use apidoc directive in extending-akka.md (#22904)

### PR DESCRIPTION
**Pull request by Captain1653**
References #22904

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by Captain1653** [comment]
Can it be reviewed in 2.6.18 milestone?) Please :)

**Review by patriknw — APPROVED** [review]
LGTM, with a small suggestion

File: akka-docs/src/main/paradox/extending-akka.md [hunk 1]
```diff
@@ -83,7 +84,7 @@ Java
 ## Applicability
 
 The sky is the limit!
-By the way, did you know that Akka `Cluster`, `Serialization` and other features are implemented as Akka Extensions?
+By the way, did you know that Akka @ref:[Cluster](typed/cluster.md), @ref:[Serialization](serialization.md) and other features are implemented as Akka Extensions?
```
[the remark this answers is not in the document]
Comment (Captain1653) [reply akka-docs/src/main/paradox/extending-akka.md]: Thank you @patriknw . It's done .

**Comment by patriknw** [comment]
thank you

**Pull request merged**